tests/drum/conftest.py

```diff
@@ -0,0 +1,236 @@
+    (R_FIT, RDS): os.path.join(TRAINING_TEMPLATES_PATH, "r_lang"),
+    (PYTHON, PYTORCH): os.path.join(TRAINING_TEMPLATES_PATH, "python3_pytorch"),
+    (PYTHON, SKLEARN_ANOMALY): os.path.join(TRAINING_TEMPLATES_PATH, "python3_anomaly_detection"),
+}
+
+targets = {BINARY: "Species", REGRESSION: "MEDV"}
+
+_class_labels = {
+    (SKLEARN, BINARY): ["Iris-setosa", "Iris-versicolor"],
+    (XGB, BINARY): ["Iris-setosa", "Iris-versicolor"],
+    (KERAS, BINARY): ["Iris-setosa", "Iris-versicolor"],
+    (RDS, BINARY): ["Iris-setosa", "Iris-versicolor"],
+    (PYPMML, BINARY): ["Iris-setosa", "Iris-versicolor"],
+    (CODEGEN, BINARY): ["yes", "no"],
+    (MOJO, BINARY): ["yes", "no"],
+    (POJO, BINARY): ["yes", "no"],
+}
+
+_artifacts = {
+    (None, REGRESSION): None,
+    (None, BINARY): None,
+    (SKLEARN, REGRESSION): os.path.join(TESTS_ARTIFACTS_PATH, "sklearn_reg.pkl"),
+    (SKLEARN, REGRESSION_INFERENCE): os.path.join(TESTS_ARTIFACTS_PATH, "sklearn_reg.pkl"),
+    (MULTI_ARTIFACT, REGRESSION): [
+        os.path.join(TESTS_ARTIFACTS_PATH, "sklearn_reg.pkl"),
+        os.path.join(TESTS_ARTIFACTS_PATH, "keras_reg.h5"),
+    ],
+    (CODEGEN_AND_SKLEARN, REGRESSION): [
+        os.path.join(TESTS_ARTIFACTS_PATH, "java_reg.jar"),
+        os.path.join(TESTS_ARTIFACTS_PATH, "sklearn_reg.pkl"),
+    ],
+    (SKLEARN, BINARY): os.path.join(TESTS_ARTIFACTS_PATH, "sklearn_bin.pkl"),
+    (KERAS, REGRESSION): os.path.join(TESTS_ARTIFACTS_PATH, "keras_reg.h5"),
+    (KERAS, BINARY): os.path.join(TESTS_ARTIFACTS_PATH, "keras_bin.h5"),
+    (XGB, REGRESSION): os.path.join(TESTS_ARTIFACTS_PATH, "xgb_reg.pkl"),
+    (XGB, BINARY): os.path.join(TESTS_ARTIFACTS_PATH, "xgb_bin.pkl"),
+    (PYTORCH, REGRESSION): [
+        os.path.join(TESTS_ARTIFACTS_PATH, "torch_reg.pth"),
+        os.path.join(TESTS_ARTIFACTS_PATH, "PyTorch.py"),
+    ],
+    (PYTORCH, BINARY): [
+        os.path.join(TESTS_ARTIFACTS_PATH, "torch_bin.pth"),
+        os.path.join(TESTS_ARTIFACTS_PATH, "PyTorch.py"),
+    ],
+    (RDS, REGRESSION): os.path.join(TESTS_ARTIFACTS_PATH, "r_reg.rds"),
+    (RDS, BINARY): os.path.join(TESTS_ARTIFACTS_PATH, "r_bin.rds"),
+    (CODEGEN, REGRESSION): os.path.join(TESTS_ARTIFACTS_PATH, "java_reg.jar"),
+    (CODEGEN, BINARY): os.path.join(TESTS_ARTIFACTS_PATH, "java_bin.jar"),
+    (POJO, REGRESSION): os.path.join(
+        TESTS_ARTIFACTS_PATH,
+        "pojo_reg",
+        "drf_887c2e5b_0941_40b7_ae26_cae274c4b424.java",
+    ),
+    (POJO, BINARY): os.path.join(
+        TESTS_ARTIFACTS_PATH,
+        "pojo_bin",
+        "XGBoost_grid__1_AutoML_20200717_163214_model_159.java",
+    ),
+    (MOJO, REGRESSION): os.path.join(TESTS_ARTIFACTS_PATH, "mojo_reg.zip"),
+    (MOJO, BINARY): os.path.join(TESTS_ARTIFACTS_PATH, "mojo_bin.zip"),
+    (PYPMML, REGRESSION): os.path.join(TESTS_ARTIFACTS_PATH, "iris_reg.pmml"),
+    (PYPMML, BINARY): os.path.join(TESTS_ARTIFACTS_PATH, "iris_bin.pmml"),
+}
+
+custom = {
```

Comment:
I'm not sure what this variable name means. maybe custom_file_location?

Reply:
thanks, fixed